src/App.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import './App.css';
+import { Box, Typography } from '@mui/joy';
+import ResponsiveAppBar from './components/ResponsiveAppBar';
+import CompoundForm from './components/form/CompoundForm';
+import { formatSums } from './utils/helpers';
+import SumsCard from './components/ui/SumsCard';
+//-------------------------------------------------
+// interface ParentProps {
+//   sendDataToParent: (data: {
+//     futureValue: number;
+//     totalInterest: number;
+//     futureValueArray: number[];
+//   }) => void;
+// }
+//-------------------------------------------------
+
+function App() {
+  const [futureValue, setFutureValue] = useState<number | string>('');
+  const [totalInteres, setTotalInterest] = useState<number | string>('');
+  const [totalDeposits, setTotalDeposits] = useState<number | string>('');
+  const [futureValueArray, setfutureValueArray] = useState<number[]>([]);
+  const [years, setYears] = useState<number>(0);
+
+  const [submited, setSubmited] = useState<boolean>(false);
+
+
+  function handleFormData(data: {
```

Comment:
```
{
     futureValue: number;
     totalInterest: number;
     futureValueArray: number[];
     yearsNum: number;
   }
```
this can be defined as a type:
```
type FormData = {
     futureValue: number;
     totalInterest: number;
     futureValueArray: number[];
     yearsNum: number;
   }
```
and then used like this:
`function handleFormData(data: FormData)`